### PR TITLE
Integration/2.0.2

### PR DESCRIPTION
**Pull request by sobyte**

### CODE REVIEW

**Comment by benjohnde** [comment]
What about #25?

**Comment by sobyte** [comment]
> What about #25?

Is now included in this PR. Thanks @benjohnde !

**Comment by benjohnde** [comment]
Awesome! Looking forward to the release of these changes :) 🚀 !

File: CHANGELOG_en-GB.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# 2.0.2
+* Customers are now being updated in the Unzer Insight Board
+* Fixed backwards compatibility to Shopware 6.3 and lower for SEPA payment methods
+* Fixed the maximum birthday year
+* Fixed the order list override so other plugins are also able to manipulate the list
+* Compatibiity with showpare 6.4.3.0 ensured
```
Comment (benjohnde) [inline CHANGELOG_en-GB.md:6]: Two typos: `Compatibility` and `Shopware`

**Review by benjohnde — APPROVED** [review]
LGTM, only found a small typo in the changelog!

File: CHANGELOG_en-GB.md (hunk 1 above)
Comment (sobyte) [reply CHANGELOG_en-GB.md]: Good catch. I've fixed these. Thanks :)

**Comment by benjohnde** [comment]
Is there any ETA for this PR? We would like to bump our shop to Shopware 6.4.3.1.

**Comment by rlincke** [comment]
Will this also work with Shopware 6.4.4.1?

**Pull request closed without merging**